fw/Core/Hitcon/App/TamaApp.cc

```diff
@@ -461,6 +466,18 @@ void TamaApp::XbRoutine(void* unused) {
     }
     // We need to know enemy score to update our frames
     UpdateFrameBuffer();
+    my_assert(_enemy_score);
+    my_assert(_enemy_score->packet_type ==
+              TAMA_XBOARD_PACKET_TYPE::PACKET_SCORE);
+    // Send result with TwoBadgeActivity
+    hitcon::game::TwoBadgeActivity activity = {
+        .gameType = hitcon::game::EventType::kTama,
+        .myScore = _qte_score,
+        .otherScore = _enemy_score->score,
+        .nonce = _my_nounce + _enemy_score->nonce,
+    };
+    memcpy(activity.otherUser, _enemy_score->user, sizeof(_enemy_score->user));
```

Comment:
nit: We should provably maintain a copy of the packet instead of tracking the pointer of enemy score. I don't think there will be any concurrency issues here though